android/app/build.gradle

```diff
@@ -0,0 +1,75 @@
+plugins {
+    id 'com.android.application'
+    id 'org.jetbrains.kotlin.android'
+}
+
+def PACKAGE_NAME = 'au.com.robin.sms'
```

Comment:
You've created folders `au/com/robin/sms` already, so if I decide to change `PACKAGE_NAME`, does it mean I have to change those folders as well?

Reply:
I'm fine with keeping the folder as such, but having `PACKAGE_NAME` changeable without changing the folders would be good.